samples/dynamically_add_module_sample/src/main.c

```diff
@@ -11,49 +11,102 @@ int main(int argc, char** argv)
     if (argc != 3)
     {
         printf("usage: dynamically_add_module_sample modulesConfigFile linksConfigFile\n");
-        printf("where configFile is the name of the file that contains the Gateway configuration\n");
+        printf("where modulesConfigFile is the name of the file that contains gateway modules and linksConfigFile is the name of the file that contains gateway links.\n");
+        printf("press ENTER to exit.\n");
```

Comment:
I disagree, specially on Visual Studio, it's good to wait to leave when something fails. It's why I made this change on this sample. 

---
In reply to: [108970369](https://github.com/Azure/azure-iot-gateway-sdk/pull/189#discussion_r108970369) [](ancestors = 108970369)